### PR TITLE
Package versions upgrading.

### PR DESCRIPTION
**Pull request by HavierD**
In Dependabot, some alerts are not able to solve (related to word-wrap, semver, and tough-cookie). It because that the react-scripts newest version 5.0.1 is using the earlier version of other sub-dependencies.

Node version specified in dockerfile.build & dockerfile are also upgraded.

Some packages have main version upgrade for solving alerts (including axios and jest/dom). May have unexpected issues in the future, but currently, I didn't come across errors.

@mui/style has deprecated and stop supporting react 17+ (we are using react 18). New ticket for it has created for next iteration.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,24 +1,39 @@
...
 ENV TZ="Australia" \
-    NVM_DIR="/opt/nvm" \
     MAVEN_HOME="/opt/maven" \
     MAVEN_VERSION=3.8.5 \
-    NODE_VERSION=16.16.0 \
+    NODE_MAJOR=18 \
     HOME="/home/builder" \
     JAVA_TOOL_OPTIONS="-Duser.home=/home/builder"
 
 RUN yum install --quiet --assumeyes git python3 python3-pip apache-maven shadow-utils tar
```
Comment (vietnguyengit) [inline Dockerfile:12]: remove this one, `ubuntu` doesn't use `yum`

Comment (vietnguyengit) [reply Dockerfile:12]: like discussed, use animal tracking files, but install bump2version for NRMN

**Comment by utas-raymondng** [comment]
Do no merge it after PR, some testing is going on in edge.

**Comment by vietnguyengit** [comment]
> Do no merge it after PR, some testing is going on in edge.

yep no merge any time soon @utas-raymondng , looks like @HavierD still have a bunch of stuff to fix, I just make sure Docker stuff is fine for him to keep going on.

@HavierD see one of my commits, temporarily commented out complaining lines that deprecated  , you need to turn it back on LOCALLY, so you can debug and fix.

**Comment by HavierD** [comment]
I have completed all work for this ticket. Please let me know when it can be merged, so that i can make the state of the pull request from DRAFT to READY.

**Comment by vietnguyengit** [comment]
next steps after all commits that to mark to `READY` then wait until official review on your PR by other devs. And move your backlog item into PR column

**Comment by utas-raymondng** [comment]
> In Dependabot, some alerts are not able to solve (related to word-wrap, semver, and tough-cookie). It because that the react-scripts newest version 5.0.1 is using the earlier version of other sub-dependencies.
> 
> Node version specified in dockerfile.build & dockerfile are also upgraded.
> 
> Some packages have main version upgrade for solving alerts (including axios and jest/dom). May have unexpected issues in the future, but currently, I didn't come across errors.
> 
> @mui/style has deprecated and stop supporting react 17+ (we are using react 18). New ticket for it has created for next iteration.

Please include the new ticket url here thanks

**Review by utas-raymondng — APPROVED** [review]
LGTM

**Pull request merged**